.gitvote.yml

```diff
@@ -0,0 +1,263 @@
+# GitVote Configuration for Hiero Governance Repository
+# This configuration allows voting within specific maintainer groups
+# Usage: /vote profile=<team-name>-maintainers-vote
+
+automation:
```

Comment:
is it possible to define this on org level (like in .github repo)? While votes will happen in government repo to add new committers/maintainer teams might need voting on other topics that should happen in the government repo:

- votes on new releases (if needed)
- votes on issues / prs / ... (if needed)